app-backend/ingest/src/main/scala/com/azavea/rf/ingest/model/OutputDefinition.scala

```diff
@@ -12,20 +17,24 @@ import java.net.URI
   * @param crs              The CRS of the projection to target in tiling
   * @param cellType         A GeoTrellis CellType to be used in storing a tile layer
   * @param cellSize         The expected 'native resolution' size of each cell
-  * @param histogramProjects The number of bins with which to construct histograms used in coloring
+  * @param histogramBuckets The number of bins with which to construct histograms used in coloring
```

Comment:
In rebase I had to rename this back to `histogramBuckets` from `histogramProjects`. This refers to the number of bins in the histogram and not the application idea of a bucket. Changing this back does not appear to impact anything else does that sounds right @Lknechtli ?